config/locales/en.yml

```diff
@@ -369,6 +369,10 @@ en:
               not_empty: should be blank if the request is not complete
             request_type:
               blank: cannot be blank
+        data_request_area:
+          attributes:
+            data_request_area_type:
+              blank: Select where the data you are requesting is from
```

Comment:
Is "Select what type of data you are requesting" better? At the moment it sounds like it's asking for the location.